GiftedMessenger.js

```diff
@@ -2,6 +2,7 @@ import React, {
   Component,
 } from 'react';
 import {
+  ActivityIndicatorIOS,
```

Comment:
it is better to use a cross platform component like - https://github.com/FaridSafi/react-native-gifted-spinner

`ActivityIndicatorIOS` will be replace by `ActivityIndicator` on RN0.28